themes/src/main/resources/theme/base/login/login-password.ftl

```diff
@@ -33,11 +39,12 @@
                     </div>
 
                     <div id="kc-form-buttons" class="${properties.kcFormGroupClass!}">
-                    <input tabindex="4" class="${properties.kcButtonClass!} ${properties.kcButtonPrimaryClass!} ${properties.kcButtonBlockClass!} ${properties.kcButtonLargeClass!}" name="login" id="kc-login" type="submit" value="${msg("doLogIn")}"/>
-                  </div>
-            </form>
+                        <input tabindex="4" class="${properties.kcButtonClass!} ${properties.kcButtonPrimaryClass!} ${properties.kcButtonBlockClass!} ${properties.kcButtonLargeClass!}" name="login" id="kc-login" type="submit" value="${msg("doLogIn")}"/>
+                    </div>
+                </form>
+            </div>
         </div>
-      </div>
+        <script type="text/javascript" src="${url.resourcesPath}/js/passwordVisibility.js"></script>
```

Comment:
Since loading scripts through `<script type="module">` is now [widely supported](https://caniuse.com/es6-module), I'd prefer we use that. It has several [distinct advantages](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#other_differences_between_modules_and_standard_scripts).

Reply:
You can check the new commit